src/webloader.js

```diff
@@ -149,32 +158,23 @@ function main() {
 		return;
 	}
 
-	if (!evolv.store) {
-		evolv.store = function (key, value, session) {
-			if (script.dataset.evolvUseCookies && !session) {
-				const domain = script.dataset.evolvUseCookies === 'true' ? "" : script.dataset.evolvUseCookies;
-				return setCookie('evolv:' + key, value, 365, domain);
-			}
-			(session ? window.sessionStorage : window.localStorage).setItem('evolv:' + key, value);
-		}
-	}
+	let storageManager = new EvolvStorageManager(script.dataset.evolvUseCookies, !requireConsent(script));
 
-	if (!evolv.retrieve) {
-		window.evolv.retrieve = function (key, session) {
-			if (script.dataset.evolvUseCookies && !session) {
-				return getCookie('evolv:' + key);
-			}
-			return (session ? window.sessionStorage : window.localStorage).getItem('evolv:' + key);
-		}
-	}
+  if (!evolv.store) {
+    evolv.store = storageManager.store;
```

Comment:
There's no functional difference with this presently because `store()` doesn't reference `this` at the moment. But it could in the future.
```suggestion
    evolv.store = storageManager.store.bind(storageManager);
```